<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new QSO</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="notesEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>243</y>
     <width>461</width>
     <height>95</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="notesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>205</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>456</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="dateTimeLabel">
      <property name="text">
       <string>Date/time</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="powerLabel">
      <property name="text">
       <string>Power TX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="callSignLabel">
      <property name="text">
       <string>Call sign</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="callSignEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="bandLabel">
      <property name="text">
       <string>Band</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="bandEdit">
      <item>
       <property name="text">
        <string>70cm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>160m</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="frequencyLabel">
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="modeLabel">
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="modeEdit">
      <item>
       <property name="text">
        <string>CW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FM</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="frequencyEdit">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>1.810000000000000</double>
      </property>
      <property name="maximum">
       <double>440.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="powerTxEdit">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>2000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>143</y>
     <width>461</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="signalTxLabel">
      <property name="text">
       <string>Signal TX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="signalTxEdit">
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="signalRxLabel">
      <property name="text">
       <string>Signal RX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="signalRxEdit">
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="gridTxLabel">
      <property name="text">
       <string>Grid TX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="gridTxEdit">
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="gridRxLabel">
      <property name="text">
       <string>Grid RX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="gridRxEdit">
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
